docs/hydra/guides/deploy-hydra-example.mdx

```diff
@@ -0,0 +1,494 @@
+```
+
+Install Ory Hydra:
+
+<CodeBlock className="language-shell">{`
+cd /opt/hydra/bin
+# Download a new version of Ory Hydra
+wget https://github.com/ory/hydra/releases/download/${useLatestRelease('hydra')}/hydra_${useLatestRelease(
+  'hydra'
+)}-linux_64bit.tar.gz
+# Extract contents
+tar xfvz hydra_${useLatestRelease('hydra')}-linux_64bit.tar.gz
+# Remove redundant files
+rm *md
+rm LICENSE
+`}</CodeBlock>
+
+Download the [Quickstart](../5min-tutorial) configuration files:
+
+<CodeBlock className="language-shell">{`cd ../config
+wget https://raw.githubusercontent.com/ory/hydra/${useLatestRelease('hydra')}/contrib/quickstart/5-min/hydra.yml`}</CodeBlock>
+
+We need to add some configuration in `hydra.yml` to use our Postgres We need configure Ory Hydra to use the Postgres database.
+Open `hydra.yml` and change the dsn configuration. In the default quickstart configuration Ory Hydra is storing data in an SQLite
+database. Change the dsn key to use the Postgres database you configured before and add the `issuer` URL. This URL will be used as
+`issuer` in access and ID tokens.
+
+```yml
+dsn: postgres://hydra:b0qw68gr3Q@127.0.0.1:5432/hydra?sslmode=disable&max_conns=20&max_idle_conns=4
+urls:
+  self:
+    issuer: https://oauth2.example.com
+```
+
+Apply [migrations](../cli/hydra-migrate-sql):
+
+```bash
+/opt/hydra/bin/hydra -c /opt/hydra/config/hydra.yml migrate sql -y postgres://hydra:b0qw68gr3Q@127.0.0.1:5432/hydra?sslmode=disable
+```
+
+Test your setup using the [`serve` command](../cli/hydra-serve):
+
+```bash
+/opt/hydra/bin/hydra -c /opt/hydra/config/hydra.yml serve all
+```
+
+You should see something like this once the service has been started:
+
+```bash
+WARN[2022-04-15T15:24:29Z] JSON Web Key Set "hydra.openid.id-token" does not exist yet, generating new key pair...  audience=application service_name=Ory Hydra service_version=v1.11.7
+WARN[2022-04-15T15:24:34Z] JSON Web Key Set "hydra.jwt.access-token" does not exist yet, generating new key pair...  audience=application service_name=Ory Hydra service_version=v1.11.7
+Thank you for using Ory Hydra v1.11.7!
+
+Take security seriously and subscribe to the Ory Security Newsletter. Stay on top of new patches and security insights.
+
+>> Subscribe now: http://eepurl.com/di390P <<
+INFO[2022-04-15T15:24:37Z] Software quality assurance features are enabled. Learn more at: https://www.ory.sh/docs/ecosystem/sqa  audience=application service_name=Ory Hydra service_version=v1.11.7
+WARN[2022-04-15T15:24:37Z] JSON Web Key Set "hydra.https-tls" does not exist yet, generating new key pair...  audience=application service_name=Ory Hydra service_version=v1.11.7
+INFO[2022-04-15T15:24:41Z] Setting up http server on :4444               audience=application service_name=Ory Hydra service_version=v1.11.7
+INFO[2022-04-15T15:24:41Z] Setting up http server on :4445               audience=application service_name=Ory Hydra service_version=v1.11.7
+```
+
+## Run Ory Hydra using systemd
```

Comment:
It'd make sense to turn this paragraph into an ordered list to show the flow better.